workshops/templates/workshops/event_create_form.html

```diff
@@ -1,5 +1,12 @@
 {% extends "workshops/_page.html" %}
 
+{% load crispy_forms_tags %}
+{% load selectable_tags %}
+
+{% block extrastyle %}
+  {% include_ui_theme %}
+{% endblock %}
+
```

Comment:
If we move loading tags to the top of the template, we can move `{% load breadcrumbs %}`, too.